tests/stringable.dora

```diff
@@ -0,0 +1,13 @@
+fun main() {
+  str[Bool](true);
+  str[Bool](false);
+  str[Byte](1.toByte());
+  str[Int](1);
+  str[Long](1L);
+  // thread 'main' panicked at 'fp-register accessed as gp-register.'
```

Comment:
Is this still a bug here?

Reply:
No, shouldn't be, I'll update the commit.